### DISCUSSION TITLE
Duda Test Cases "REPORTE-PLANETAS" mayuscula/minuscula

### DISCUSSION THREAD

**Discussion by AndrewVenegas** [IIC2133-PUC/2024-1#47] [Tarea 0]
Hola!

Solamente quería avisar que en los testcases entregados, la palabra "pedidos" se encuentra en minúsculas. Frente a ello, quería asegurarme de que tenga que ser en minúsculas, y no en mayusculas como ha sido en casi todos los outputs de la tarea.

Quedo atento!

Saludos :D

**Comment by benjavicente** [comment]
Hola y perdón por la demora!

Se respeta lo que sale en el enunciado, pero como es un problema _**muy**_ menor, **evitaremos descontar por eso**. Si es que alguien cree que hay un problema por ese caso, se puede apelar y obtener el puntaje correspondiente sin problemas.
